rosdep/base.yaml

```diff
@@ -4389,6 +4389,7 @@ libogre-1.12-dev:
 libogre-dev:
   arch: [ogre-1.9]
   debian:
+    bookworm: [libogre-1.12-dev]
```

Comment:
Looking at this more closely, I think this is actually incorrect.  While it isn't explicit in the name of this key, this is implicitly ogre 1.9.  Since ogre 1.9 is not available in bookworm, I don't think we should add this here.

If you'd like to use libogre-1.12-dev, I'm going to suggest that we add in a new key called `libogre-1.12-dev`.  I don't love doing that, but I think it is the most consistent thing to do here.